frontend/src/locales/index.tsx

```diff
@@ -1,15 +1,10 @@
-import { setupI18n } from '@lingui/core'
-import { Catalogs, I18n } from '@lingui/core/i18n'
+import { i18n } from '@lingui/core'
 
 /**
  * To create a new language or translation, for example, `en-custom`:
- * `npm run add-locale en-custom`
- * then import it here
- *  */
+ * Add it to the `locales` array in `.linguirc` and run `npm run extract`
+ */
 
 import catalogEn from './en/messages.js'
-const catalogs: Catalogs = { en: catalogEn }
-export const i18n: I18n = setupI18n({
-  language: 'en', // language to display for the site. Currently this is hard-coded and cannot be changed via UI.
-  catalogs,
-})
+i18n.load('en', catalogEn.messages)
```

Comment:
```suggestion
import { en } from 'make-plural/plurals'

i18n.loadLocaleData('en', { plurals: en })
i18n.load('en', catalogEn.messages)
```

There's a warning in the console for `Locale data for locale "en" not loaded. Plurals won't work correctly.`. Although we don't use plurals (yet), we should probably load it.

Reply:
Oops, I'll properly commit this